### PR TITLE
30/css

### PR DESCRIPTION
**Pull request by pyphilia**

### CODE REVIEW

File: src/services/export.js [hunk 1]
```diff
@@ -291,18 +296,25 @@ const saveEpub = async (page, interactive) => {
...
 
   // get body for epub
   // use the export class to differentiate from tools content
   const body = await page.$$eval('.export > section', phases =>
-    phases.map(phase => ({
-      title: phase.getElementsByClassName('name')[0].innerHTML,
-      data: phase.getElementsByClassName('resources')[0].innerHTML,
-    }))
+    phases.map(phase => {
+      const content = phase.getElementsByClassName('resources')[0].innerHTML;
```
Comment (juancarlosfarah) [inline src/services/export.js]: We should factor out the strings `resources` and `description` to the selectors file.

File: src/services/export.js [hunk 2]
```diff
@@ -291,18 +296,25 @@ const saveEpub = async (page, interactive) => {
...
+
+      let description = phase.getElementsByClassName('description')[0];
+      description = description ? description.outerHTML : '';
+
+      return {
+        title: phase.getElementsByClassName('name')[0].innerHTML,
```
Comment (juancarlosfarah) [inline src/services/export.js]: same with `name`

File: src/services/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
+* {
```
Comment (juancarlosfarah) [inline src/services/css/styles.css]: Can we remove the unnecessary commented out bits from this file?

File: src/services/export.js (hunk 1 above)
Comment (pyphilia) [reply src/services/export.js]: I refactor them in the other pull request.

File: src/services/export.js (hunk 2 above)
Comment (pyphilia) [reply src/services/export.js]: same as above

**Comment by juancarlosfarah** [comment]
@pyphilia, OK, sounds good. However, it looks like there are conflicts now. Can you please resolve them? Thanks!

**Comment by juancarlosfarah** [comment]
Hi @pyphilia, please remember to fix the conflicts here so that I can merge.

**Comment by juancarlosfarah** [comment]
@pyphilia,  this PR still has conflicts.

**Pull request merged**